nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-processors/src/test/java/org/apache/nifi/processors/kafka/KafkaPublisherTest.java

```diff
@@ -124,33 +112,9 @@ public void validateSuccessfulSendAsDelimited() throws Exception {
     }
 
     @Test
-    public void validateSuccessfulSendAsDelimited2() throws Exception {
```

Comment:
No, that was actually an addition when we tried to use Scanner from java that would parse using regex, so need to test for special character, but since that approach never seen the light of day that test was obsolete from the get go.